.github/workflows/ci.yml

```diff
@@ -1,46 +1,56 @@
 name: CI
 
 on:
-  push:
-    branches: [ main ]
   pull_request:
-    branches: [ main ]
+    branches: [main, beta]
+    types: [opened, reopened, synchronize, edited]
 
 jobs:
-  test:
-    name: Test
+  audit:
```

Comment:
```suggestion
  audit:
    name: Audit
```

Reply:
These are nits, but I'm trying to make this prettier:
![image](https://user-images.githubusercontent.com/7625337/236577916-a43732d4-9c4f-4e51-a8f7-d9bd5d7650b5.png)

Reply:
✅